css/styles.css

```
@@ -1,20 +1,345 @@
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 100%;
+  justify-content: center;
+}
+
+.input__group {
+  display: flex;
+  flex-direction: column;
+}
+
+.input__group label {
+  font-size: 1.2rem;
+  font-weight: bold;
+  margin-bottom: 8px;
+  opacity: 0.7
+}
+
+.form__flex-wrap .input__group label {
+  text-align: center;
+  margin-left: 0px;
+  margin-bottom: 8px;
+  margin-right: 32px;
+}
+
+.input__form i {
+  text-align: center;
+  display: block;
+  font-style: normal;
+  color: #d1d1d1;
+  opacity: 0.9;
+  font-size: 0.9rem;
+  margin: 8px auto;
+}
+
+.input__form>.form__row input {
+  margin: 0px auto;
+}
+
+input {
+  border: 0px solid white;
+  border-radius: 40px;
+  padding: 16px 24px;
+  font-size: 2rem;
+  font-family: inherit;
+  color: inherit;
+  font-weight: bold;
+  transition: 0.2s all;
+  margin: 8px 0px;
+  margin-right: 32px;
+}
+
+input[type=number]:placeholder-shown {
+  background: #f3f3f3;
 }
 
-#turnipTable thead tr:first-child {
-    border: none;
+input[type=number]:not(:placeholder-shown) {
+  background: transparent;
+  color: #0AB5CD;
+}
+
+input[type=number]:focus {
+  outline: none;
+  transform: scale(1.1);
+  background: white;
+  box-shadow: 0 1px 6px rgba(0, 0, 0, 0.04), 0 3px 6px rgba(0, 0, 0, 0.07);
+}
+
+input[type=number]:focus::placeholder {
+  opacity: 0;
+}
+
+input[type=number] {
+  width: 60px;
+  text-align: center;
+}
+
+input::-webkit-outer-spin-button,
+input::-webkit-inner-spin-button {
+  -webkit-appearance: none;
+  margin: 0;
+}
+
+input[type=number] {
+  -moz-appearance: textfield;
+}
+
+.reset-button {
+  font-family: inherit;
+  font-weight: bold;
+  color: #FFBABA;
+  padding: 8px 16px;
+  border-width: 0px;
+  border-radius: 40px;
+  background: transparent;
+  font-size: 1.2rem;
+  transition: 0.2s all;
+  position: relative;
+  margin: 32px auto;
+  opacity: 0.5;
+}
+
+.reset-button:hover {
+  cursor: pointer;
+  opacity: 1;
+  box-shadow: 0 1px 6px rgba(0, 0, 0, 0.04), 0 3px 6px rgba(0, 0, 0, 0.07);
+}
+
+
+.table-wrapper {
+  width: 98%;
+  max-height: 50vh;
+  padding: 16px;
+  margin: 0px auto;
+  box-sizing: border-box;
+  overflow-x: scroll;
+  border-radius: 2px;
+}
+
+#turnipTable {
+  border-collapse: collapse;
+}
+
+#turnipTable span {
+  min-width: 80px;
 }
 
 #turnipTable th {
-    text-align: center;
+  min-width: 80px;
+  font-size: 0.8rem;
 }
 
-#turnipTable td:first-child {
-    white-space: nowrap;
-    text-align: left;
+#turnipTable th div:nth-of-type(1) {
+  margin-bottom: 2px;
+}
+
+#turnipTable th div:nth-of-type(2) {
+  display: flex;
+  justify-content: space-around;
+  opacity: 0.4;
 }
 
 #turnipTable td {
-    text-align: center;
+  min-width: 80px;
+  max-width: 84px;
+  padding: 4px 2px;
+  text-align: center;
+  font-size: 0.8rem;
+  border-right: 1px solid rgba(0, 0, 0, 0.03);
+  border-bottom: 1px solid rgba(0, 0, 0, 0.15);
+}
+
+#turnipTable tbody tr {
+  opacity: 0.6;
+}
+
+#turnipTable tbody tr:hover {
+  cursor: default;
+  opacity: 1;
+}
+
+
```

```suggestion
.table-wrapper::-webkit-scrollbar {
  height: 8px;
}

.table-wrapper::-webkit-scrollbar-track {
  height: 8px;
  -webkit-box-shadow: inset 0 0 6px rgba(0,0,0,0.2);
}

.table-wrapper::-webkit-scrollbar-thumb {
  height: 8px;
  background: rgba(0,0,0,0.2);
  -webkit-box-shadow: inset 0 0 6px rgba(0,0,0,0.1);
}
.table-wrapper::-webkit-scrollbar-thumb:window-inactive {
  height: 8px;
  background: rgba(0,0,0,0.2);
}
```

Styles the scroll bar on narrow windows based systems on supported browsers.